<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>currencysChangeDialog</class>
 <widget class="QDialog" name="currencysChangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="currenciesNameLineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="currenciesAltName">
      <property name="inputMask">
       <string>AAA; </string>
      </property>
      <property name="cursorPosition">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="currenciesNameLabel">
      <property name="text">
       <string>CyrrensyName</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="currenciesAltLabel">
      <property name="text">
       <string>CyrrensyAltName</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="currenciesRateLineEdit">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="inputMask">
       <string>99999999.99; </string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="maxLength">
       <number>11</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="currenciesRateLabel">
      <property name="text">
       <string>CurrensyRate</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="currenciesNationalCurrensyLabel">
      <property name="text">
       <string>NationalCurrensy</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="currenciesNationalCurrensyCheckBox">
      <property name="text">
       <string>National</string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="currenciesCodeLineEdit">
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="inputMask">
       <string>999999; </string>
      </property>
      <property name="cursorPosition">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="currenciesCodeLabel">
      <property name="text">
       <string>Code</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="currenciesButtonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="currenciesIdLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="currenciesIdLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>147</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>currenciesNameLineEdit</tabstop>
  <tabstop>currenciesAltName</tabstop>
  <tabstop>currenciesRateLineEdit</tabstop>
  <tabstop>currenciesCodeLineEdit</tabstop>
  <tabstop>currenciesNationalCurrensyCheckBox</tabstop>
  <tabstop>currenciesButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>currenciesButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>currencysChangeDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>currensiesNew()</slot>
 </slots>
</ui>
